Exporter/src/main/java/uk/ac/ebi/biostd/exporter/jobs/full/json/JsonPublicOnlySubmissionProcessor.java

```diff
@@ -0,0 +1,30 @@
+package uk.ac.ebi.biostd.exporter.jobs.full.json;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.collect.Lists;
+import org.easybatch.core.processor.RecordProcessor;
+import org.easybatch.core.record.Record;
+import org.springframework.stereotype.Component;
+import uk.ac.ebi.biostd.exporter.jobs.full.model.StringContentRecord;
+import uk.ac.ebi.biostd.exporter.model.Submission;
+
+@Component
+public class JsonPublicOnlySubmissionProcessor implements RecordProcessor<Record<Submission>, Record<String>> {
+    private final ObjectMapper objectMapper;
+
+    public JsonPublicOnlySubmissionProcessor(ObjectMapper objectMapper) {
+        this.objectMapper = objectMapper;
+    }
+
+    @Override
+    public Record processRecord(Record<Submission> record) throws Exception {
+        if (record.getPayload() instanceof Submission) {
+            Submission submission = record.getPayload().clone();
```

Comment:
I think is simpler to add lombok `@Builder(toBuilder = true)` to class  and then `submission.toBuilder().accessTags(emptyList()).build()` in this way you do not need to copy manually

Reply:
Done, nice catch